### DISCUSSION TITLE
npm run dev shows error: Watchpack error (watcher): EISDIR: illegal operation on a directory, readlink <PROJECT PATH>

### DISCUSSION THREAD

**Discussion by alec330** [vercel/next.js#59786] [Help]
### Summary

Hello all, I am new to this community, so not sure if I am asking at the right place, but Please help if you have experienced this issue. When I first created a directory installing next.js using npx create-next-app@latest ., I was running npm run dev and it seems to have failed to compile; showing error in the additional info. I am currently running on Ubuntu in VSC, coding in Typescript

I am stuck here as I have went to stack overflow and didn't have luck. I am pretty new to fullstack development so please let me know if any of you could help! 

Thanks!!

### Additional information

```js
WatchPack Error (watcher): Error: EISDIR: illegal operation on a directory, readlink <PROJECT PATH>
```

**Comment by inkhaton** [comment]
i get this too - but only if i run my angular app from a shared folder in virtualbox
